clippy_lints/src/matches.rs

```diff
@@ -229,6 +229,40 @@ declare_clippy_lint! {
     "a wildcard enum match arm using `_`"
 }
 
+declare_clippy_lint! {
+    /// **What it does:** Checks for wildcard enum matches for a single variant.
+    ///
+    /// **Why is this bad?** New enum variants added by library updates can be missed.
+    ///
+    /// **Known problems:** Suggested replacements may not use correct path to enum
+    /// if it's not present in the current scope.
```

Comment:
If that is the case, please make the suggestion `Applicability::MaybeIncorrect`

Reply:
I've took it from wildcard_enum_match_arm. Should I fix there too?

Reply:
I've applied the suggestion to both of them for now